config/metadata/rulesMeta.yaml

```diff
@@ -609,7 +609,24 @@ groups:
         description: >
           The field to check. This can name any field in the trace. If the field
           is not present, then the condition will not match. The comparison is
-          case-sensitive.
+          case-sensitive. 
+          
+          The field can also include a prefix that changes the span used for evaluation of the field.
+          The only prefix currently supported is `root`, as in `root.http.status`. Specifying `root.` 
+          causes the condition to be evaluated against the root span. 
+          
+          For example:
+          
+          Field: root.url
```

Comment:
can you please add code markers around this section? That's something that `suggest` doesn't support.

Reply:
Yes, good callout!